seed/views/labels.py

```diff
@@ -218,9 +222,6 @@ def add_labels(self, qs, inventory_type, inventory_ids, add_label_ids):
             model = self.models[inventory_type]
             inventory_model = self.inventory_models[inventory_type]
             exclude = self.exclude(qs, inventory_type, add_label_ids)
-            inventory_ids = inventory_ids if inventory_ids else [
```

Comment:
This was just a reassignment, so I just removed it. Otherwise, all code changes mirror this: https://github.com/SEED-platform/seed/commit/e5e149d4809e3870582a39b2a91906e848c09679